doc/check.texi

```diff
@@ -974,6 +974,7 @@ easier for the developer to write, run, and analyze tests.
 * Test Fixtures::               
 * Multiple Suites in one SRunner::  
 * Selective Running of Tests::
+* Selecting Tests Based on Arbitrary Tags::
```

Comment:
You'll need to add:
```
* Selecting Tests by Suite or Test Case
```
here. Otherwise, the index is the following:
```
...
4.5 Multiple Suites in one SRunner	  	
4.6 Selective Running of Tests	  	
4.6.2 Selecting Tests Based on Arbitrary Tags	  	
4.7 Testing Signal Handling and Exit Values	
...  
```